compiler/z/codegen/BinaryEvaluator.cpp

```diff
@@ -2476,56 +2504,75 @@ genericLongAndAsRotateHelper(TR::Node * node, TR::CodeGenerator * cg)
       TR::Node * skipedConversion = NULL;
```

Comment:
There doesn't seem to be any uses of this variable. Are we able to fold it away?